src/test/java/com/example/amazoncom/amazontest.java

```diff
@@ -7,51 +7,38 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import java.time.Duration;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class amazontest {
     public WebDriver driver;
+    AmazonMainPage mainPage;
 
+    final static String WEB="https://www.amazon.es/";
     @BeforeAll
     public void setupBrowser() {
-        System.out.println("Test starts in amazon.es");
         WebDriverManager.chromedriver().setup();
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.addArguments("incognito");
+        //chromeOptions.addArguments("headless");
         driver = new ChromeDriver(chromeOptions);
-        driver.manage().window().maximize();
-        driver.get("https://www.amazon.es/");
-        this.driver = driver;
-        WebElement waiting = (new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//input[contains(@id,'twotabsearchtextbox')]")))));
+        driver.get(WEB);
```

Comment:
pls, open the page via the function defined on the page object level.

Reply:
and I strongly recommend you to use wait for element displayed when you open the page before continue your test, to avoid "element not found" exception.